### DISCUSSION TITLE
Default Thread Admin Password in Thread Commissioning Application (andriod)

### DISCUSSION THREAD

**Discussion by 3rang** [openthread/openthread#8284] [Q&A] [answered]
Hi 

I am trying to use the Andriod app for commissioning, but I don't know what password need to use to connect BR. 

setup RPI 4b  + nrf52840 DK (Thread device ) . all services are up and running. 

I tried with `ot-ctl dataset pskc -p 123456` (but didn't work)

any hint .!!

**Comment by jwhui** [comment, marked as answer]
The commissioner passphrase is configured when forming the Thread network.

The [External Commissioning](https://openthread.io/guides/border-router/external-commissioning) guide provides more information.
